rotkehlchen/chain/ethereum/modules/eigenlayer/balances.py

```diff
@@ -162,6 +162,9 @@ def _query_token_pending_withdrawals(self, balances: 'BalancesSheetType') -> 'Ba
         )
         for address, event_list in addresses_with_withdrawals.items():
             for event in event_list:
+                if event.asset == A_ETH:
```

Comment:
All of the PR is re-adding what I removed before. Except this. This is the actual fix that should have been done.

We count all in-flight withdrawals from here by checking withdrawal events. For native restaking we also check the eigenpod. So for the native ETH asset we should not count the withdrawals.